content/finalist/Voomp/index.md

```diff
@@ -0,0 +1,66 @@
+---
+type: "finalist"
+title: Team Voomp
+name: "Voomp"
+description: "We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone."
+Team-Quote: "Driving forward with technology"
+Application_Type: "Mobile Application"
+---
+
+We at Voomp believe technology enhances every aspect of life and aim make it accessible to everyone.
+
+---
+
+# Project's Name
+
+_**Theme-Name**_ : AtmaNirbhar Bharat
+
+_**Application-Type**_ : Mobile Application
+
+---
+
+# Description
+
+In light of recent events, we saw the government of India banning several Chinese operated applications. A few of them were UC News, NewsDog, and QQ Newsfeed, these were known to be India's top content (news) aggregrator and distributor through their content recemmendation engine. We at Voomp, aim to fill this void by creating India's own content aggregation and distribution platform.
+
+We plan to develop a fully custom backend which will serve news articles. The backend will also support fliter based on positive/negitave sentiment of the content.
+
+The platform will consist majorly of 3 parts:
+
+1. Aggregation Service, which will search and store data from various sources.
+2. Distribution Service, this will be responsible for serving the content through APIs. Similar to newsapi.org, this will enable 3rd party apps to use our backend for content.
+3. Android application, this is the frontend through which user will be able to consume the content.
+
+---
+
+_**Tech-Stack**_ : Android, Python, Django, PostgreSQL, Docker, AWS
+
+_**GitHub-Link**_ : https://github.com/SubhamPramanik/voomp
+
+_**Hosted-Link**_ : https://drive.google.com/file/d/1lfFNbnTIMi_HxJ0-HyrX7ICiGcC-PRDH/view?usp=sharing and http://ec2-15-207-58-167.ap-south-1.compute.amazonaws.com/api/articles
```

Comment:
What is this ? Separate the link and also mention which link is for what?

Reply:
Resolved